controllers/UserController.js

```diff
@@ -0,0 +1,100 @@
+import UserModel from '../models/User.js'
+import jwt from 'jsonwebtoken'
+import bcrypt from 'bcrypt'
+import { validationResult } from 'express-validator';
+
```

Comment:
You can extract the part of the code that generates the token into a separate function to avoid code duplication:
```suggestion
const generateToken = (user) => {
    return jwt.sign(
        {
            _id: user._id,
        },
        process.env.ID_KEY,
        {
            expiresIn: '30d'
        }
    );
}
```